codalab/lib/worksheet_util.py

```diff
@@ -1008,6 +1008,7 @@ def flush_worksheets():
                     raw_to_block.append((len(blocks) - 1, 0))
             elif item_type == TYPE_DIRECTIVE:
                 command = get_command(value_obj)
+                raw_to_block_has_append = False
```

Comment:
What does `raw_to_block_has_append` mean? I wonder if we should name it to something more clear...

Reply:
It means whether the `raw_to_block` var has executed the `append` operation. So that we can decide whether we should do the  `append` operation or not later. If you think it is not clear, do you have any advice on it?

Reply:
How about `raw_to_block_need_append`?